app/renderer/components/sectionTitle.js

```diff
@@ -0,0 +1,131 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const React = require('react')
+const ImmutableComponent = require('../../../js/components/immutableComponent')
+
+const {StyleSheet, css} = require('aphrodite/no-important')
+const globalStyles = require('./styles/global')
+const commonStyles = require('./styles/commonStyles')
+
+class SectionTitleWrapper extends ImmutableComponent {
```

Comment:
I endorse creating a new folder to include this file but despite being enthusiastic on creating small components, they're still too small to host their own files IMO. 

Since they're just titles, it's very unlikely that we'll add more methods and bulk file with new code.

For the new folder name, I'd suggest something like common/shared to host this, and in future include things like `button` and our already existent `settings` component, so for all highly replicated components. Even if we decide to create an `about` folder -- which I endorse as well -- they should be on that folder